eks/cleanup.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+ 
+set +e
+
+## Dop cleanup
+helm uninstall kubera 
```

Comment:
I find following to be more readable. Whats your take?
```sh
if [[ -f "flux.yaml" && $CLEAN == true ]]; then
    echo -e "\nCleaning up manifests."
    echo "Set CLEAN=false if you wish for this not to occur."
    rm -rf flux.yaml
fi
```

Reply:
@AmitKumarDas what is `flux.yaml` here

Reply:
@rohan2794 I was referring to code syntax & not flux